cellmlmanip/model.py

```diff
@@ -410,10 +410,17 @@ def get_equation_graph(self, refresh=False) -> nx.DiGraph:
         self.graph = graph
         return graph
 
-    def get_equations_for(self, symbols):
-        """Get all equations for given collection of symbols"""
+    def get_equations_for(self, symbols, lexicographical_sort=True):
+        """Get all equations for given collection of symbols
+
+        Results are sorted in topographical order.
+        Lexicographical_sort indicates whether the result is sorted in lexicographical order first
```

Comment:
I think we're using `:param lexicographical_sort: indicates...` elsewhere?

Reply:
good point although shouldn't symbols also have that?